kubernetes/opentelemetry-demo.yaml

```diff
@@ -10093,6 +10093,8 @@ spec:
           - containerPort: 8080
             name: service
           env:
+          - name: FEATURE_FLAG_GRPC_SERVICE_ADDR
```

Comment:
This file is generated after from the Helm chart, after we do a release in that repo. Please revert the changes in this file.